mlflow/server/js/src/experiment-tracking/components/ArtifactPage.js

```diff
@@ -140,7 +156,10 @@ export class ArtifactPageImpl extends Component {
 
   render() {
     return (
-      <RequestStateWrapper requestIds={this.listArtifactRequestIds}>
+      <RequestStateWrapper
+        requestIds={this.listArtifactRequestIds}
+        // eslint-disable-next-line no-trailing-spaces
```

Comment:
@xanderwebs In our private repo, this part looks like this:

```suggestion
        // eslint-disable-next-line no-trailing-spaces
        description={SomeClass.VARIABLE_STORING_STRING}
```

Reply:
I think we can remove this.